logstash-core/lib/logstash/api/commands/base.rb

```diff
@@ -8,22 +8,18 @@ def initialize(service = LogStash::Api::Service.instance)
           @service = service
         end
 
-        def hostname
-          service.agent.node_name
-        end
-
         def uptime
           service.agent.uptime
         end
-
+        
```

Comment:
My editor put it in but maybe it makes sense? White space should be indented too!